CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
-
+- [cygnus-common] [sql] Fix sql syntax for expiration records (select, delete, filters) in non mysql instances
+- [cygnus-common] [sql] Exclude error table from persist policy (cap records is based on recvtime which is non used by persistError)
+- [cygnus-common] [sql] Add missed capRecords implementation for PSQL backends
```

Comment:
If this PR is a fix for issue #2242, that issue should be mentioned in CNR (not sure if applies to the three lines or only to a subset of them).